web/modules/custom/do_base/do_base.deploy.php

```
@@ -765,6 +851,257 @@ function _do_base_menu_leading_weight(string $menu_name): int {
+ *
+ * @param \Drupal\node\NodeInterface $node
+ *   The node to write to.
+ * @param array $tags
+ *   Metatag values keyed by tag name.
+ */
+function _do_base_set_seo_metatags(NodeInterface $node, array $tags): void {
+  if (empty($tags) || !$node->hasField('field_n_metatags')) {
+    return;
+  }
+
+  $current = Json::decode((string) $node->get('field_n_metatags')->value) ?: [];
```

PHPMD (2.15.0)

[error] Avoid using static access to class '\Drupal\Component\Serialization\Json' in method '_do_base_set_seo_metatags'. (undefined)

(StaticAccess)